uk.co.compucorp.civicrm.hrcore/CRM/HRCore/APIWrapper/DefaultLimitRemover.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+class CRM_HRCore_APIWrapper_DefaultLimitRemover implements API_Wrapper {
+
+  /**
+   * In civicrm API, 0 means there is no limit on the retrieved results
+   */
+  const NO_LIMIT_ON_RESULTS = 0;
+
+  /**
+   * the wrapper contains a method that allows you to alter the parameters of the api request (including the action and the entity)
```

Comment:
I tend to leave default civi docblocks for such things , but sure .. I think it's better to reformat the default docblocks